### PR TITLE
I added a Shop for Box! And some minor changes Part 2

### PR DESCRIPTION
**Pull request by AlphaBuffalo**
I added a small shop for all your fencing needs. As well as fixed some minor things not really worth mentioning because it was literally just adding an air alarm in places that didnt have it, and changing tiles to fit my autistic needs.

One thing I did change was the wiring, basically, the station's wiring was in the shape of a U and most people can cut it at one point and cause a blackout to 1 half of the station. I now changed it so you have to cut 2 wires, 3 at most. It helps with explosions as well so engineers have more time on fixing the breach since they dont have to worry about losing power in part of the station, though, the shuttle will be called anyways :\

These are some of the new wires I placed

### CODE REVIEW

**Comment by ultraplen** [comment]
copy pasting my comment from the other pr.

the shop layout is absolute garbage tbh. a vox shuttle style layout would be better.

**Comment by AlphaBuffalo** [comment]
Im not very good at mapping... If someone else wants to do something about they can, I dont care.

File: maps/vgstation.dmm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"a" = (/turf/space,/area)
```
Comment (ghost) [inline maps/vgstation.dmm:1]: ????

Comment (Exxion) [inline maps/vgstation.dmm:1]: IT'S BACK

Comment (AlphaBuffalo) [inline maps/vgstation.dmm:1]: Should I just delete that file? It serves no purpose from what it looks like. Its literally an empty map file.

**Comment by kernikov** [comment]
I noticed you didn't add a new / use a different ID access, hope you like greytiders breaking in your back door and stealing stuff.

**Comment by Exxion** [comment]
The Deff shop doesn't have special access either, and it works just fine
You just need to attach a signaler to the bolt wire
That makes it roughly equally hard to break into as any other door you don't have access to

But in the other PR, he said he was thinking about adding a new access for it

**Comment by kernikov** [comment]
How many people use the deff shop?
I've never seen it used.

File: maps/vgstation.dmm (hunk 1 above)
Comment (Exxion) [inline maps/vgstation.dmm:1]: If it actually is completely empty, I don't see why not. The only line in the entire codebase that references the file is commented out, so it shouldn't break anything.

**Comment by Exxion** [comment]
I've used it, and I think I've seen it used one other time
It's a shame it's not used more often, as it's pretty good if you do the signaler thing

**Comment by PJB3005** [comment]
`code\game\jobs\access.dm:268:error: acces_shop: undefined var`

**Comment by AlphaBuffalo** [comment]
The new access that was added is needed to open those windoors and the two tables on the top are displays cases

File: code/game/jobs/access.dm [hunk 2]
```diff
@@ -264,7 +265,7 @@
 		if(5) //command
 			return list(access_heads, access_RC_announce, access_keycard_auth, access_change_ids, access_ai_upload, access_teleporter, access_eva, access_tcomsat, access_gateway, access_all_personal_lockers, access_heads_vault, access_hop, access_captain)
 		if(6) //station general
-			return list(access_kitchen,access_bar, access_hydroponics, access_janitor, access_chapel_office, access_crematorium, access_library, access_theatre, access_lawyer, access_clown, access_mime)
+			return list(acces_shop, access_kitchen,access_bar, access_hydroponics, access_janitor, access_chapel_office, access_crematorium, access_library, access_theatre, access_lawyer, access_clown, access_mime)
```
Comment (Exxion) [inline code/game/jobs/access.dm]: `acces_shop`
You missed an s

**Comment by AlphaBuffalo** [comment]
thanks PJB for helping me

File: code/game/jobs/access.dm (hunk 2 above)
Comment (AlphaBuffalo) [inline code/game/jobs/access.dm]: shhhhhhhhhhhh no I didnt

Comment (Exxion) [inline code/game/jobs/access.dm]: Oh, you fixed this before I made the comment but GitHub didn't tell me that at the time
Sorry

**Comment by kernikov** [comment]
Is this a round start shop or is it worthless until someone asks the HoP/steals the spare?

**Comment by AlphaBuffalo** [comment]
Someone has to ask the HoP/steal the spare to use it. I didnt want it so people can just open it because there are no access requirements

**Comment by AlphaBuffalo** [comment]
How did I earn the Balance Label?

**Comment by AlphaBuffalo** [comment]
whoops

**Comment by 9600bauds** [comment]
> One thing I did change was the wiring, basically, the station's wiring was in the shape of a U and most people can cut it at one point and cause a blackout to 1 half of the station. I now changed it so you have to cut 2 wires, 3 at most. It helps with explosions as well so engineers have more time on fixing the breach since they dont have to worry about losing power in part of the station

This is a big balance change

**Comment by ComicIronic** [comment]
Yeah, that.

**Comment by AlphaBuffalo** [comment]
Oh yeah, the shop was an after thought that took precedence over the actual reason of the pr... Welp

**Comment by NigglyWiggly** [comment]
I'd remove the shop, to be honest.
If people want to have a little shop they'll typically make one themselves, which is a lot more fun than just having a space set up just for that to begin with.

**Comment by AlphaBuffalo** [comment]
I'll probably still add the shop access if that's the case. But it wouldn't bother me if that's the better route

**Comment by AlphaBuffalo** [comment]
But one of the benefits to something like this is that you can save time, when I do assistant shop I normally have to try 3 rounds because of wizards, ops, and so on. Not to mention security giving you shit for repurposing the aux tool storage, and other people breaking in because there isn't much stopping them.

**Comment by ComicIronic** [comment]
So you made a big balance change, and on top of that you decided to de-atomicise this PR with a content change as well?

This PR is breaking two requirements:
- It isn't atomic
- It makes a major balance change without consulting the community.

**Comment by AlphaBuffalo** [comment]
Fair enough I'll separate it, could have just said that instead of calling me atomic and being rude. This is why people dont want to add content because the rules aren't clearly laid out and I was in the IRC the entire time refering to the people in there and no one said anything like this. Good job on teaching me the rules though.

**Comment by ComicIronic** [comment]
The guidelines are clearly shown to you when you try to open a PR. Both those rules are in there. Moreover, Github tells you to check the repo's contribution guidelines https://github.com/d3athrow/vgstation13/blob/FeatureFreeze/CONTRIBUTING.md which also contains these rules.

I can't apologise if you think closing this PR is rude. The rules are laid out very clearly.

**Comment by IratePirate** [comment]
Can't say i'm a fan. Breaks up maintenance and isn't very good looking.

**Comment by 9600bauds** [comment]
Heya, sorry about the mix-up. Everyone screws up at first, it can be a little disconcerting to have a PR of yours closed. You're more than welcome to try again, shame that this happened in a map PR where you can't easily separate the content like you could separate code. Hope to see more from you, good luck.

**Comment by AlphaBuffalo** [comment]
I wasn't expect the PR to merge, I'm still learning and all, but it was that instead of saying hey I cant merge this because its 2 changes in one. I get told it cant get merged because it is atomic. 

Though thanks for the feedback on the shop and stuff, if I can find another place to add it later I will and we'll see about merging it then.

**Comment by Exxion** [comment]
1. The PR can't be merged because it's NON-atomic. Atomic is good.
2. Changing two unrelated things at once is exactly what non-atomic means.

Did you think atomic was some sort of personal insult?

**Comment by ComicIronic** [comment]
Fucking atoms, shut them down

**Comment by ghost** [comment]
ComicAtomic

**Comment by AlphaBuffalo** [comment]
I did think it was an insult but I see what you mean now

**Pull request closed without merging**